src/tests/postgres/compose.yaml

```diff
@@ -2,7 +2,12 @@ services:
   x:
     image: postgres
     x-defang-postgres:
-
+      maintenance-window: Mon:00:00-Mon:01:00
```

Comment:
What timezone is this? 
Shall we split this into start/stop begin/end or something like that? To avoid creating our own funky types/parsing and we can stick to regular timestamps (also solves TZ problem)

Reply:
UTC by default but I get your point. We allow users to specify their own TZ elsewhere we should allow that here